.github/workflows/push.yml

```
@@ -26,13 +26,21 @@ jobs:
         uses: docker/login-action@v1
         with:
           registry: ghcr.io
-          username: ${{ secrets.GHCR_DOCKER_USERNAME }}
-          password: ${{ secrets.GHCR_DOCKER_TOKEN }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: test image exists
+        run: |
+         mkdir -p ~/.docker
+         echo '{"experimental": "enabled"}' >> ~/.docker/config.json
+         echo "IMAGE_EXISTS=$(docker manifest inspect ghcr.io/${{ github.repository }}:${{ github.sha }} > /dev/null && echo 1 || echo 0)" >> $GITHUB_ENV
```

also the tennens must be included in the name
```suggestion
         echo "IMAGE_EXISTS=$(docker manifest inspect ghcr.io/${{ github.repository }}-${{ matrix.tennens }}:${{ github.sha }} > /dev/null && echo 1 || echo 0)" >> $GITHUB_ENV
```